package.json

```
@@ -61,6 +61,6 @@
     "eslint": "^7.0.0"
   },
   "engines": {
-    "node": "^10.12.0 || >=12.0.0"
+    "node": "12.* || 14.* || >= 16.*"
```

I think we should bring them in line with ESLint 8.0.0

```suggestion
    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```

---

ESLint raised the requirement to these higher versions mostly so they could use ESM code/dependencies.

I'm open to using these versions for eslint-plugin-eslint-plugin, but I'm not aware of a specific need/benefit from us doing so at this time.

If we converted eslint-plugin-eslint-plugin to ESM soon, which could be a fairly-large task if it's even practical right now, that would be a good reason to raise the versions. But we could also do this in a subsequent major version if we don't get to it now.

---

agreed. and the plugin supports multi eslint versions - does not have to be in line with newest eslint.

---

Yeah, I think I would rather we allow all Node 12/14/16 users to use this linting plugin, until we eventually convert it to ESM. So I'll leave the code as-is.